src/wai_data_tools/manual_labeling.py

```diff
@@ -1,13 +1,12 @@
-"""This module allows manual labeling by users."""
+"""Module for manually relabeling frames using GUI."""
```

Comment:
maybe splitting the business logic and the GUI would provide us better ability to automate it later on if needed